src/components/sections/home/Banner.tsx

```diff
@@ -80,29 +81,16 @@ const Banner: React.FC = () => {
                 </Link>
               </div>
             </BannerWrapper>
-            <svg
-              width="24"
-              height="24"
-              viewBox="0 0 24 24"
-              fill="none"
-              xmlns="http://www.w3.org/2000/svg"
-              onClick={() => setBannerOpen(!bannerOpen)}
-            >
-              <path
-                d="M18.75 5.25L5.25 18.75"
-                stroke="#7E8F9A"
-                stroke-width="2"
-                stroke-linecap="round"
-                stroke-linejoin="round"
+            <span className="closeBtn">
+              <StaticImage
```

Comment:
you can use a normal img tag...as this is just an svg...gatsby can't optimise this further